Cargo.toml

```diff
@@ -17,6 +17,7 @@ crossterm = { version = "0.26.1", features = ["use-dev-tty"] }
 ratatui = "0.21.0"
 regex = { version = "1.8.2", default-features = false, features = ["std", "perf", "unicode-case" ] }
 arboard = { version = "3.2.0", default-features = false, features = ["wayland-data-control"] }
+termbg = "0.4.3"
```

Comment:
This bring in async-std and a lot of other deps. We don't even need any async. Considering that currently `cargo install` is the main method people can get gimoji (except on Fedora), this is problematic. Is there no other way to detect this?

What's the impact on a clean build time?

Reply:
Updated to use [terminal-light](https://github.com/Canop/terminal-light) instead which has lighter dependencies and still seems to work, at least in kitty, gnome-console (both light) and gnome-terminal (dark theme) as well as xterm (default white bg).

Build time hovers around the 20s mark on my machine with/without this commit now.

Reply:
I build and tested locally and it looks good. :+1: